test/parser/postgresparser_test.cpp

```diff
@@ -724,5 +724,23 @@ TEST_F(PostgresParserTests, CreateDbTest) {
   delete stmt_list;
 }
 
+TEST_F(PostgresParserTests, CreatePGSQLFuncTest) {
+  std::string query = "CREATE OR REPLACE FUNCTION increment(i integer) RETURNS integer AS $$   BEGIN  RET  j;  END; $$ LANGUAGE plpgsql;";
```

Comment:
Just a sample UDF. We just support a preliminary parse as of now, that does not validate the syntax completely. This level validation only happens when the function is actually executed.